package.json

```diff
@@ -19,37 +19,38 @@
     "get-cert": "./scripts/get_cert_fingerprint.sh"
   },
   "dependencies": {
-    "@celo/connect": "1.5.2",
-    "@celo/contractkit": "1.5.2",
-    "@celo/utils": "1.5.2",
-    "@celo/wallet-hsm-aws": "1.5.2",
-    "@celo/wallet-hsm-azure": "1.5.2",
-    "bignumber.js": "^8.1.1",
+    "@celo/connect": "5.2.0",
+    "@celo/contractkit": "7.1.0",
+    "@celo/utils": "6.0.0",
+    "@celo/wallet-hsm-aws": "5.1.3",
+    "@celo/wallet-hsm-azure": "5.1.3",
+    "bignumber.js": "^9.0.0",
     "bunyan": "1.8.15",
     "express": "^4.17.1",
     "js-yaml": "^3.13.1",
     "mathjs": "^7.5.1",
     "node-fetch": "^2.6.1",
     "prom-client": "^12.0.0",
     "tslint-no-focused-test": "^0.5.0",
-    "web3": "1.3.6"
+    "web3": "1.10.0",
+    "web3-core": "1.10.0",
+    "web3-core-subscriptions": "1.10.0",
+    "web3-eth": "1.10.0"
   },
   "devDependencies": {
-    "@celo/typescript": "^0.0.1",
+    "@celo/typescript": "^0.0.2",
     "@types/bunyan": "^1.8.6",
     "@types/express": "^4.17.6",
     "@types/jest": "28.1.2",
     "@types/mathjs": "^6.0.5",
     "@types/node-fetch": "^2.5.5",
-    "jest": "28.1.2",
+    "jest": "29.7.0",
     "prettier": "2.1.1",
-    "ts-jest": "^28.0.5",
+    "ts-jest": "^29.1.2",
     "tslint": "^6.1.1",
     "tslint-config-prettier": "^1.18.0",
     "tslint-eslint-rules": "^5.4.0",
     "tslint-microsoft-contrib": "^6.2.0",
-    "tslint-react": "^4.2.0",
```

Comment:
lol why were these ever there

Reply:
I also wondered about that 😆

Reply:
Mento Oracle UI leak confirmed wait for airdrop 😂